api/v1/db/oracledb/games-dao.js

```diff
@@ -0,0 +1,47 @@
+const appRoot = require('app-root-path');
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+
+/**
+ * @summary Return a list of games
+ * @function
+ * @returns {Promise<Object[]>} Promise object represents a list of games
+ */
+const getGames = async (queries) => {
+  // parse passed in parameters and construct query
+  const sqlParams = {
+    scoreMin: parseFloat(queries.scoreMin),
+    scoreMax: parseFloat(queries.scoreMax),
+  };
```

Comment:
I don't think you need to parse `queries.scoreMin` and `queries.scoreMax` to floating number manually since the express-openapi should convert it automatically when loading the openapi file: https://github.com/osu-mist/game-curator-api/blob/master/openapi.yaml#L448.
In addition, when applying score parameters, all game with the null score still present. Is it expected?

Reply:
You're right I don't need the parses. I wasn't really sure how I wanted to handle nulls so I just left them in. I changed it so that when a min or max score is passed in the nulls no longer show.